solr/core/src/test/org/apache/solr/cluster/events/ClusterEventProducerTest.java

```diff
@@ -102,7 +105,7 @@ public void teardown() throws Exception {
 
   @Test
   public void testEvents() throws Exception {
-    int version = waitForVersionChange(-1, 10);
+    int version = versionTracker.waitForVersionChange(-1, 10);
```

Comment:
Version starts at 0 in the implementation of the version tracker, so what does it mean to wait for it to be -1?

Reply:
Maybe we need a `getCurrentVersion` in `VersionTrackerImpl` so a given instance can be reused more easily in a given test?

Reply:
-1 is the last value we've "seen", I'll add some docs around this. Effectively this is a getVersion in this implementation.